client/my-sites/site-settings/composing/after-the-deadline.jsx

```diff
@@ -173,7 +173,7 @@ class AfterTheDeadline extends Component {
 		} = this.props;
 
 		return (
-			<FormFieldset>
+			<FormFieldset className="site-settings__has-divider is-top-only">
```

Comment:
Right now there is a double border between the `JetpackMarkdown` and the `AfterTheDeadline` settings:

![](https://cldup.com/YR4WoFPlGf.png)